nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/pom.xml

```diff
@@ -134,11 +133,26 @@
             <artifactId>bval-jsr</artifactId>
             <version>2.0.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.codehaus.jackson</groupId>
+            <artifactId>jackson-mapper-asl</artifactId>
+            <version>1.9.13</version>
+        </dependency>
```

Comment:
Is this dependency still required with the refactored classes that use Jackson 2?